src/main/java/alfio/manager/TicketReservationManager.java

```diff
@@ -925,13 +929,25 @@ public static boolean hasValidOfflinePaymentWaitingPeriod(PaymentContext context
      *
      * @param paymentMethodDTO
      * @param purchaseContext
-     * @param configurationManager
      * @return
      */
-    public static boolean isValidPaymentMethod(PaymentManager.PaymentMethodDTO paymentMethodDTO, PurchaseContext purchaseContext, ConfigurationManager configurationManager) {
+    public boolean isValidPaymentMethod(PaymentManager.PaymentMethodDTO paymentMethodDTO, PurchaseContext purchaseContext) {
+        var ifCustomOfflineIsApplicableForEvent = true;
```

Comment:
we should declare this variable as `boolean ifCustomOfflineIsApplicableForEvent = true` to avoid unnecessary (un)boxing